packages/popup/src/hooks/index.js

```diff
@@ -121,12 +121,19 @@ export const useEstimateTx = (tx = {}, tokensAmount = {}) => {
     loading,
     error,
   } = useAsync(async () => {
-    if (!provider || !currentNetwork?.netId || (!to && !data)) return
+    if (
+      !provider ||
+      !currentNetwork?.netId ||
+      (!to && !data) ||
+      !network.chainId
+    )
+      return
     return await estimate(tx, {
       type,
       request: provider.request.bind(provider),
       tokensAmount,
       isFluentRequest: true,
+      chainIdToGasBuffer: {[network.chainId]: network.gasBuffer},
```

Comment:
`chainIdToGasBuffer: network.gasBuffer` chainIdToGasBuffer can be an integer `gasBuffer` here. Then need not call ''eth_chainId` in estimate package. Why need pass `chainId`?

Reply:
1. there's **no way** to get the `gasBuffer` value without calling RPC to get the current `chainId`
2. caller can use the `defaultGasBuffer` field to do what you sugges

Reply:
Got it~